netbout-client/src/main/java/com/netbout/mock/MkBase.java

```diff
@@ -122,5 +110,26 @@ public Bout randomBout() throws IOException {
         );
         return bout;
     }
+    /**
+     * Create alias.
+     * @param name Name of the alias
+     * @return Alias
+     * @throws IOException If fails
+     */
+    public Alias alias(final String name) throws IOException {
```

Comment:
@pinaf thank you for comments, i removed that method and refactored test to use existing method, it is better now.